hash/risc0/bench/README.md

```diff
@@ -0,0 +1,101 @@
+# RISC Zero Rust Starter Template
+
+Welcome to the RISC Zero Rust Starter Template! This template is intended to
+give you a starting point for building a project using the RISC Zero zkVM.
```

Comment:
Starter template README is not that useful. Edit down to the useful parts (no need for Bonsai for example), and add relevant information, like what this code does?